### PR TITLE
Create Portal Login page #7

### PR DESCRIPTION
**Pull request by YogenRaii**

### CODE REVIEW

File: src/views/CreateAccount/CreateAccount.js [hunk 1]
```diff
@@ -54,15 +56,27 @@ export default class CreateAccount extends React.Component {
     this.setState({ confirmPassword: e.target.value });
   }
 
+  getErrorMessage() {
+    if (this.state.error !== "") {
```
Comment (YogenRaii) [inline src/views/CreateAccount/CreateAccount.js:60]: @rrsampathi can we do do `if(this.state.error)` which will return false if `error` is `null, undefined, ""` etc?

File: src/views/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import Logo from '../../assets/images/Respond_Logo_icon_fullcolor.png'
+import Footer from "../../components/Footer/Footer"
+import './Login.css';
+
+export default class CreateAccount extends React.Component {
```
Comment (YogenRaii) [inline src/views/Login/Login.js:6]: @rrsampathi  is it type between CreateAccount or LoginPage?

File: src/views/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    updatePassword(e) {
+      this.setState({password: e.target.value})
+    }
+
+    onChangeCheckbox(e) {
+      this.setState({rememberMe: e.target.checked})
+    }
+
+    getErrorMessage() {
+      if (this.state.error !== "") {
```
Comment (YogenRaii) [inline src/views/Login/Login.js:42]: same as above

File: src/views/CreateAccount/CreateAccount.js (hunk 1 above)
Comment (rrsampathi) [reply src/views/CreateAccount/CreateAccount.js:60]: Yup that should work.

File: src/views/Login/Login.js (hunk 3 above)
Comment (rrsampathi) [reply src/views/Login/Login.js:42]: Yup, same change should work.

File: src/views/Login/Login.js (hunk 2 above)
Comment (rrsampathi) [reply src/views/Login/Login.js:6]: Ah sorry, forgot to switch the class name to Login. Good catch!

**Pull request merged**